snapcraft_legacy/internal/meta/_manifest.py

```diff
@@ -28,10 +29,20 @@
     from snapcraft_legacy.project import Project
 
 
+def _format_datetime(dt: datetime) -> str:
+    """Format a datetime object into a string compatible with the store.
+
+    :param dt: The datetime object to format.
+
+    :return: A string representing the datetime.
+    """
+    return dt.strftime("%Y-%m-%dT%H:%M:%S.%fZ")
+
+
 def annotate_snapcraft(project: "Project", data: Dict[str, Any]) -> Dict[str, Any]:
     manifest = OrderedDict()  # type: Dict[str, Any]
     manifest["snapcraft-version"] = snapcraft_legacy._get_version()
-    manifest["snapcraft-started-at"] = project._get_start_time().isoformat() + "Z"
+    manifest["snapcraft-started-at"] = _format_datetime(project._get_start_time())
```

Comment:
Surely just the ISO time code is good enough?

Reply:
Almost certainly, but I'm being cautious and formatting it exactly as it previously was formatted.